src/logic/bitwise.rs

```diff
@@ -30,91 +16,60 @@ impl ApInt {
 
     /// Flip all bits of this `ApInt` inplace.
     pub fn bitnot(&mut self) {
-        self.modify_digits(|digit| digit.not_inplace());
+        self.modify_digits(|digit| digit.flip_all());
         self.clear_unused_bits();
     }
 
     /// Tries to bit-and assign this `ApInt` inplace to `rhs`
     /// and returns the result.
-    /// 
```

Comment:
Could you please re-add all `# Error` sections in the doc-comments? They are actually part of the general Rust documentation guidelines.